Runtime/Internal/DotNetWebSocket.cs

```diff
@@ -264,13 +264,21 @@ private async Task SendAsync()
         {
             while (_socket.State == System.Net.WebSockets.WebSocketState.Open)
             {
-                WebSocketMessage message;
+                WebSocketMessage message = null;
+                int count;
                 lock (_outgoingMessages)
                 {
-                    if (_outgoingMessages.Count == 0)
-                        continue;
+                    count = _outgoingMessages.Count;
```

Comment:
Just wondering if there's a reason you're avoiding the continue inside the lock? As opposed to doing this:

```
if (_outgoingMessage.Count == 0)
{
    await Task.Delay(10);
    continue;
}
```

Reply:
Nevermind, I didn't look at this closely enough, I know the answer! Deleting these edits, but dropping this message for posterity.